### DISCUSSION TITLE
I can't submit my application to GitHub Student Developer Pack even though I'm a studant!

### DISCUSSION THREAD

**Discussion by DanteRael** [community/community#67950] [GitHub Education]
### Hi everyone,

I am having problems applying to GitHub Student Developer Pack. I'm currently styding Software Engineering in a EAD college. Due to being an EAD college, We don't have a student ID. I have a document that proves I'm studying currently though. I send It to Git's application page and I get this problem: 
Your application cannot be reviewed until you fix the following:
Your image should be one of the following:
Dated school ID.
I can't use an Dated school ID profile photo when my college doesn't provides school ID. Please I need help with It because I'm currently styding programming and I use Git as a professional portfolio, which helps me A LOT. So I saw this student developer pack and It is really going to help my in my studies. I don't know where else to look for help anymore.
![2023-09-22_09-00](https://github.com/community/community/assets/128925828/ffc0dd23-49f3-45c6-b40b-e404ec63eb51)

**Comment by walidbosso** [comment]
Did you find a solution?

**Comment by orobertofc** [comment]
Same problem here. I do have an id and all the documents, but they are digital, and the page doesn't allow digital documents
